api/lib/JSONValidator.js

```diff
@@ -0,0 +1,26 @@
+const Ajv = require('ajv');
+const ajv = new Ajv({allErrors: true});
+
+const Schemas = require('./schemas');
+const errorMessages = require('./error-messages');
+
+function JSON(){
+    this.validSchemas = Object.keys(Schemas);
+}
+
+JSON.prototype.isValid = function(document, schema) {
```

Comment:
This shouldn't give us problems, but since we are going to update this class anyway, what do you think of using `doc` instead of `document`, [for the same reason](https://developer.mozilla.org/en-US/docs/Web/API/Document)? (I know there's no DOM in node, but I personally feel more comfortable using a word that has no relation with a default object from front-end JS, even if it is only to avoid ambiguity. If you think it's unnecessary I'm ok too).

Reply:
I think it's good the idea to avoid any kind of ambiguity... I'll change the parameter name